CocoaMarkdown/CMImageTextAttachment.m

```diff
@@ -149,7 +149,14 @@ - (nullable NSImage *)imageForBounds:(NSRect)imageBounds textContainer:(nullable
     }    
     
 #if !TARGET_OS_IPHONE
-    [self.image setFlipped:NSGraphicsContext.currentContext.isFlipped];
+    if (! [NSProcessInfo.processInfo isOperatingSystemAtLeastVersion: (NSOperatingSystemVersion){10, 15, 0}]) {
```

Comment:
Can you change this to use @available instead?

```
if (!@available(macOS 10.15, *)) {...}
```

Reply:
Not sure it would be a good idea :

- `@available` is intended to protect APIs not available in the app deployment version. The goal is that the compiler does not issue a warning when such possibly-non-available APIs are used inside a  @available protected block.  
Btw using a negative form such as `if (!@available(…)) {…}` results in a compiler warning.

- On the other hand, `-[NSProcessInfo isOperatingSystemAtLeastVersion:]` is a perfectly-valid API, that does exactly what is needed here: implement a different behavior based on the current OS version.

Reply:
Fair enough, thanks for the explanation!